src/main/java/pl/agh/cs/io/Window.java

```diff
@@ -0,0 +1,27 @@
+package pl.agh.cs.io;
+
+public class Window {
```

Comment:
ta klasa jest przypadkiem szczególnym `WindowsPerExe`. Czy to jest konieczne?

Reply:
To jest tylko klasa na potrzeby API, nigdzie jej dalej nie eksponujemy i ułatwia pogrupowanie procesów do tege samego okna